### ISSUE TITLE
AMOS2022 LabelsTs

### ISSUE THREAD

**Issue by AN-AN-369** [MASILab/3DUX-Net#44]
Hi,thank you for your amazing work you have done on 3DUX-Net! Where did you get the labels of the test data set of AMOS2022?

**Comment by leeh43** [comment]
Hi, thank you your interest towards our work! The label that we use for AMOS 2022 evaluation is in the training set in AMOS, instead of the test set. Furthermore, I only use the CT modality for evaluation, so please take a look the .json file for task 1 in AMOS 2022 challenge.
